Cargo.toml

```diff
@@ -70,7 +70,7 @@ tokio = { workspace = true }
 
 actix-cors = "0.7.0"
 actix-files = "0.6.6"
-actix-web = { version = "4.7.0", optional = true, features = ["rustls-0_23", "actix-tls"] }
```

Comment:
how is this change related to the issue?

Reply:
No. actually I have messed up while trying to rebase

Reply:
Do you know, how to undo the changes?